Makefile

```diff
@@ -301,6 +301,18 @@ disable-mptcp:
 
 .PHONY: disable-mptcp
 
+CONFIG_FILE ?= ${HOME}/.celestia-app/config/config.toml
+SEND_RECV_RATE ?= 10485760  # 10 MiB
+
+configure-v3:
```

Comment:
is this intended to also work on mac?
If so, I can try to fix it as I am seeing this when executing:
```
Using config file at: /Users/midnight/.celestia-app/config/config.toml
sed: 1: "/Users/xxx/.celest ...": invalid command code m
sed: 1: "/Users/xxx/.celest ...": invalid command code m
sed: 1: "/Users/xxx/.celest ...": invalid command code m
make: *** [configure-v3] Error 1
```

Reply:
ahh good catch! do you mind trying this now? https://github.com/celestiaorg/celestia-app/pull/4022/commits/ad3d55d713e1f560952a0e9838fd31465bf67a01

Reply:
works, thanks 🙏 🙏